test/services/SessionServiceSpec.scala

```diff
@@ -52,7 +52,7 @@ class SessionServiceSpec extends SpecBase with MockitoSugar with BeforeAndAfter
   )
 
   before {
-    Mockito.reset(mockSessionRepository, mockEmployeeExpensesConnector)
+    Mockito.reset[Object](mockSessionRepository, mockEmployeeExpensesConnector)
```

Comment:
I find it quite odd, as I've never seen such a warning on `Mockito.reset`. It seems that in this repo, the Java Mockito is used "directly" instead of mockito-scala.

Could you try with the Scala-based dependency?